homeassistant/components/mqtt/strings.json

```
@@ -108,6 +108,110 @@
       "invalid_inclusion": "The client certificate and private key must be configured together"
     }
   },
+  "config_subentries": {
+    "device": {
+      "initiate_flow": {
+        "user": "Add MQTT Device",
+        "reconfigure": "Reconfigure MQTT Device"
+      },
+      "entry_type": "MQTT Device",
+      "step": {
+        "device": {
+          "title": "Configure MQTT device details",
+          "description": "Enter the MQTT device details:",
+          "data": {
+            "name": "[%key:common::config_flow::data::name%]",
+            "configuration_url": "Configuration URL",
+            "sw_version": "Software version",
+            "hw_version": "The hardware version of the device.",
```

I think these should be symmetric:
```suggestion
            "sw_version": "Software version",
            "hw_version": "Hardware version",
```